jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridView.java

```diff
@@ -249,9 +248,23 @@ public String getValidKeyForAdditionalListGrid( String baseName ) {
         return newListGrid;
     }
 
-    protected ExtendedPagedTable<T> createExtendedPagedTable(final GridGlobalPreferences preferences,
+    protected ExtendedPagedTable<T> createExtendedPagedTable(final GridGlobalPreferences defaultPreferences,
                                                              final String key) {
-        return TAB_SEARCH.equals(key) ? createAdvancedSearchTable(preferences) : new ExtendedPagedTable<T>(preferences);
+        GridGlobalPreferences pref;
+        ExtendedPagedTable<T> table ;
+        if(TAB_SEARCH.equals(key)){
+            pref = new GridGlobalPreferences(defaultPreferences.getKey() + key,
```

Comment:
The `if` and `else` branch have the same code for initializing `pref` - why not initialize it before if?